sdk/resource/process_test.go

```diff
@@ -102,13 +102,14 @@ func restoreAttributesProviders() {
 	resource.SetDefaultRuntimeProviders()
 	resource.SetDefaultUserProviders()
 	resource.SetDefaultOSDescriptionProvider()
+	resource.SetDefaultContainerProviders()
 }
 
 func TestWithProcessFuncsErrors(t *testing.T) {
 	mockProcessAttributesProvidersWithErrors()
 
-	t.Run("WithPID", testWithProcessExecutablePathError)
-	t.Run("WithExecutableName", testWithProcessOwnerError)
+	t.Run("WithExecutablePath", testWithProcessExecutablePathError)
+	t.Run("WithOwner", testWithProcessOwnerError)
```

Comment:
Why these changes?  They seem unrelated.

Reply:
Yeah, these are unrelated typos. I can fix these in another PR if you like.